### ISSUE TITLE
[web] text fields missing ARIA label when navigating using Tab key

### ISSUE THREAD

**Issue by yjbanov** [flutter/flutter#146854]
Text fields missing ARIA label when navigating using Tab key. In the semantics tree the text field is represented by its own input element with an ARIA label applied. However, `text_editing.dart` that kicks in when using the Tab key does not apply the ARIA label. This leads to the screen reader reading "edit, blank" with NVDA on Windows or "edit text, blank" with VoiceOver on macOS.

**Comment by yjbanov** [comment]
NVM, looks like the user was testing with semantics off.
